xbmc/cores/AudioEngine/Sinks/pipewire/AESinkPipewire.cpp

```diff
@@ -322,8 +322,8 @@ bool CAESinkPipewire::Initialize(AEAudioFormat& format, std::string& device)
 
   stream->AddListener(pipewire.get());
 
-  m_latency = 20; // ms
-  uint32_t frames = std::nearbyint((m_latency * format.m_sampleRate) / 1000.0);
+  m_latency = 20ms;
+  uint32_t frames = std::nearbyint(m_latency.count() * format.m_sampleRate);
```

Comment:
I would squash that with the fixup, as said: aim for 50 Ms period and 200 Ms buffer. Make sure the ratio of periods towards buffer is around 1 to 3 or 1 to 4

Reply:
Period == frames that are added by AE at once

Reply:
This is the also the value you set towards AE in the sink struct then. Take care of the units, please.

Reply:
Updated, the overall buffer is no 200ms, and the period is 50ms, so 4 periods.